pixart.js

```diff
@@ -1 +1,37 @@
+var button = $('#set-color')
+var brush = $('.brush')
+var colorField = $('#color-field')
+var square = $('.square')
+var first = $('#first')
+var second = $('#second')
+var third = $('#third')
+var holderColor = null
 
+button.on('click',function(){
+  setColor();
+  return false;
+})
+colorField.on('keypress', function(e){
+    if (e.keyCode == 13) {
+      setColor();
+    }
+});
+$('.history').children().on('click',function(){
+  holderColor = $(this).css('background')
+  $(this).css('background',brush.css('background'))
+  brush.css('background',holderColor)
+})
+
```

Comment:
you could also use an array to store the history on the Javascript side. It's a good practice to keep the business logic or data on the javascript side, and avoiding using the DOM as an informal kind of database.

Nice work here though!